addon/components/infinity-loader.js

```diff
@@ -28,6 +42,10 @@ const InfinityLoaderComponent = Ember.Component.extend({
     this._unbindEvent('resize');
   },
 
+  _isInfinityPromiseArray: Ember.computed('infinityModel', function() {
+    return (this.get('infinityModel') instanceof InfinityPromiseArray);
```

Comment:
Naive question - In which case is the infinityModel not an instance of InfinityPromiseArray?

Reply:
Good Q - answered below!